portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/ExposeErcFromAssetLibraryForComments.testcase

```diff
@@ -44,6 +90,39 @@ definition {
 		}
 	}
 
+	@disable-webdriver = "true"
+	@priority = "4"
+	test CanCreateCommentWithoutCustomExternalReferenceCodePreviouslyCreated {
+		property portal.acceptance = "true";
+
+		task ("And Given with POST request I create a comment without a custom external reference code") {
+			var blogPostingId = BlogPostingAPI.getBlogPostingsIdList();
+
+			var response = CommentAPI.createCommentInBlogPosting(
+				blogPostingId = "${blogPostingId}",
+				commentText = "CommentOne");
+		}
+
+		task ("When with POST request I create a comment with a custom external reference code with a value of the internal id of the previously created comment") {
+			var commentId = JSONUtil.getWithJSONPath("${response}", "$.id");
+
+			var response2 = CommentAPI.createCommentInBlogPosting(
+				blogPostingId = "${blogPostingId}",
+				commentText = "CommentTwo",
+				externalReferenceCode = "${commentId}");
+		}
+
+		task ("Then another comment with the same external reference code is being created") {
```

Comment:
As far as I know, the external reference code is unique. The reason we can create two comments is one is using UUID as default erc, and the other is using the id of previous comment. Could you help double check and update the "Then" message according? Thanks!